### PR TITLE
Use published version of pbc crates

### PR DESCRIPTION
**Pull request by spagett1maker**
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Bug fix
- [ ] Dependency, Environment, Build related update 
- [ ] Documentation
- [ ] Other tiny fix 
## Checklist before requesting a review
- [ ] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] Leave reference about this pull request, if exist (ex. discord message, google docs, etc)
- [ ] If it is a core feature, I have added thorough tests.

## Checklist after creating a pull request
- [ ] Mention reviews in discord channel directly

### CODE REVIEW

File: colony-chain/src/lib.rs [hunk 1]
```diff
@@ -1,6 +1,7 @@
 use async_trait::async_trait;
-use pbc_colony_common::*;
-use pbc_contract_common::*;
+use pdao_beacon_chain_common::message as polkadot_message;
```
Comment (junha1) [inline colony-chain/src/lib.rs:2]: 이거 왜 rename 한거예요?

**Review by junha1 — COMMENTED** [review]
`fix: Cargo ver` 이 커밋 이름 컨벤션에 맞게 바꿔주세요.

File: colony-chain/src/lib.rs (hunk 1 above)
Comment (spagett1maker) [reply colony-chain/src/lib.rs:2]: 폴카닷에서 해서 고쳐야하는걸로 판단했습니다. 충돌이라면 수정하겠습니다.

Comment (spagett1maker) [reply colony-chain/src/lib.rs]: 수정했습니다.

**Review by junha1 — APPROVED** [review]

**Pull request merged**